### PR TITLE
OKTA-729928 Add considerations for editing AIP variables in migrated OIN apps

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add considerations for editing AIP variables in migrated OIN apps
- **Is this PR related to a Monolith release?** No. But should be published after June 2024 (migration started at this point).

### Preview
https://preview-4927--reverent-murdock-829d24.netlify.app/docs/guides/submit-oin-app/openidconnect/main/#update-integration-considerations

### Resolves:

* [OKTA-729928](https://oktainc.atlassian.net/browse/OKTA-729928)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/3aa7d9e6e0aeb9c1e559a419eda2cbc41534e9bc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/694eefcc-55a2-462d-bf54-a9ab4515e726) | 85  | 88  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ecd63a3a-ab03-41db-863c-2bb2f544171e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/e668a9ee8c69064a25fe2b5319dc8a53aec4b7f7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e7eccc0b-2244-41ad-8c8e-d452bed2aaf2) | 85  | 88  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.42657048-e079-48e6-9651-e39f09373700?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/bed0adc83c2e768cc75a36224485ce6dd5a45d40/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5af13a15-f54d-4472-af21-13675bd492cf) | 84  | 88  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d3feb573-4339-492f-8efb-40538fff03e6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ykrestyanov-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ All required tests in the OIN Submission Tester must have passed within 48 hours
 
 You can modify your published SSO integration from the OIN Wizard.
 
-When you edit a published OIN integration, you need to test the SSO flow for the updated version and the published version for backwards compatibility. Testing the published version for backwards compatibility ensures that SSO to your app still works for customers who have already installed your published OIN integration. After you successfully test the updated and published versions of your integration, resubmit it to the OIN team.
+When you edit a published OIN integration, you need to test the SSO flow for the updated version and the published version for backwards compatibility. Testing the published version for backwards compatibility ensures that SSO to your app still works for customers who have already installed your published OIN integration. See [Update integration considerations](#update-integration-considerations) before you edit your published SSO integration. After you successfully test the updated and published versions of your integration, resubmit it to the OIN team.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:429]:
```suggestion
When you edit a published OIN integration, test the SSO flow for the updated version and the published version for backwards compatibility. Testing the published version for backwards compatibility ensures that SSO to your app still works for customers who have already installed your published OIN integration. See [Update integration considerations](#update-integration-considerations) before you edit your published SSO integration. After you successfully test the updated and published versions of your integration, resubmit it to the OIN team.
```

**Review by susanharper-okta — COMMENTED** [review]
1 baby nit

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/6977bf90d1b3657bac75ca671b0e22f6ed82ad98/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/73144418-7c97-4d7e-b4d4-083a527d60f9) | 84  | 88  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b815e5d9-639f-48fb-af92-db24437c41aa?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**